Добавить использование flake8 и исправить все ошибки

Прогнал код проекта через `flake8` и увидел много стилистических замечаний, которые сложно заметить самому, например:
```sh
susp/parser.py:22:5: F841 local variable 'chrome_ua' is assigned to but never used
susp/parser.py:22:80: E501 line too long (138 > 79 characters)
susp/parser.py:23:80: E501 line too long (137 > 79 characters)
susp/parser.py:46:80: E501 line too long (84 > 79 characters)
susp/parser.py:49:77: W291 trailing whitespace
```

Предлагаю внедрить использование `flake8` в проекте, исправить все текущие ошибки и в дальнейшем не мержить PR, пока не будут исправлены все стилистические замечания.

Желательно добавить в корень репы файлик `.flake8` с настройками — например, максимальную длину строки можно сделать 100 или 120, т.к. пеповские 80 уже не актуальны, на мой взгляд.

В рамках данной задачи надо будет добавить автоматическую проверку кода на соответствие `flake8` через `github actions`, с этим я помогу.

--- a/susp/.flake8
+++ b/susp/.flake8
@@ -1,0 +1,2 @@
+[flake8]
+max-line-length = 120

--- a/susp/app.py
+++ b/susp/app.py
@@ -9,10 +9,9 @@ import schedule
 
 import susp.parser
 import susp.utils
+import susp.notifications
 
 LOG = susp.utils.make_logger()
-
-import susp.notifications
 
 
 def run_threaded(job_func):

--- a/susp/event.py
+++ b/susp/event.py
@@ -4,10 +4,8 @@ from mongoengine import (
     connect,
     BooleanField,
     Document,
-    IntField,
     StringField,
 )
-from mongoengine.errors import NotUniqueError, DoesNotExist
 
 # db connection
 connect('susp', host='mongodb')

--- a/susp/notifications.py
+++ b/susp/notifications.py
@@ -37,12 +37,12 @@ def generate_event_message(event, is_new):
         message = f'Дата:[ ]({poster_url}){date}\nЦена: {price}\nОсталость мест: {seats}\n[Купить билеты]({url})'
         if is_new:
             message = 'Билеты снова в продаже!\n' + message
-            
+
         LOG.debug('Message created')
     else:
         LOG.error('Unable to generate a message: event url and poster url are missing')
         message = None
-    
+
     return message
 
 

--- a/susp/parser.py
+++ b/susp/parser.py
@@ -19,8 +19,10 @@ def get_all_events():
     """Iterates pages and returns events from all pages."""
 
     # playing with user agents
-    chrome_ua = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36"
-    safari_ua = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.2 Safari/605.1.15"
+    # chrome_ua = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/537.36 \
+    #     (KHTML, like Gecko) Chrome/39.0.2171.95 Safari/537.36"
+    safari_ua = "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/605.1.15 \
+        (KHTML, like Gecko) Version/12.1.2 Safari/605.1.15"
     headers = {"User-Agent": safari_ua}
 
     total_events = []
@@ -46,7 +48,7 @@ def get_all_events():
         # TODO: make functions for notification and saving html for further analysis
         # else:
         #     if not total_events:
-        #         notify("No events were found!")  # letting know, something 
+        #         notify("No events were found!")  # letting know, something
         #         save_html(page_html)             # for further checking
 
     return total_events
@@ -94,10 +96,10 @@ def get_event_poster_url(event):
     """Returns poster url of given event."""
 
     event_poster_url = None
-    
+
     poster_div = event.find('div', class_="js-product-img")
     event_poster_url = poster_div.get("data-original")
-    
+
     if not event_poster_url:
         poster_div = event.find('div', class_="t778__bgimg")
         event_poster_url = poster_div.get("data-original")
@@ -123,7 +125,7 @@ def get_event_availability(event):
         availibility_mapping = {
             "Нет мест": False,
             "Места есть": True,
-            "\d мест": True,
+            r"\d мест": True,
         }
         seats_div = event.find("div", class_="t778__mark")
         if seats_div:
@@ -152,7 +154,7 @@ def get_remaining_tickets(event):
     if remaining_tickets is None:
         seats_div = event.find("div", class_="t778__mark")
         if seats_div:
-            pattern = "\d мест"
+            pattern = r"\d мест"
             if re.match(pattern, seats_div.text):
                 remaining_tickets = seats_div.text.split(' ')[0]
 
@@ -238,5 +240,5 @@ def check_events():
         LOG.info(f"Events available again:  {became_available_events_count}")
 
     except Exception as e:
-        LOG.exception("While checking events exception happened:")
+        LOG.exception(f"While checking events exception happened: {e}")
         raise

--- a/susp/utils.py
+++ b/susp/utils.py
@@ -8,7 +8,7 @@ def make_logger():
     '''
     Creats and sets logger configurations.
 
-    Note: use LOG.error(message, exc_info=True) or 
+    Note: use LOG.error(message, exc_info=True) or
     LOG.exception(message) to log traceback
     '''
 

